modules/serverless-traffic-management-kn.adoc

```diff
@@ -0,0 +1,57 @@
+[id="serverless-traffic-management-kn_{context}"]
+= Splitting traffic between revisions by using the Knative CLI
+
+You can use the `kn service update` command to split traffic between revisions of a service.
+
+.Example command
+[source,terminal]
+----
+$ kn service update <service_name> --traffic <revision>=<percent>
```

Comment:
Maybe? Or just a nit picking. :)
```suggestion
$ kn service update <service_name> --traffic <revision_name>=<percent>
```

Reply:
Good thinking, but actually we explain it in the description:

* `<revision>` is the revision that you want to configure to receive a percentage of traffic. You can either specify the name of the revision, or a tag that has been applied to the revision by using the `--tag` flag.

so it doesn't need to be the name, it can also be the tag